ADAL/src/request/ADWebRequest.h

```diff
@@ -54,6 +54,7 @@ typedef void (^ADWebResponseCallback)(NSMutableDictionary *);
 }
 
 @property (strong, readonly, nonatomic) NSURL               *URL;
+@property (copy, nonatomic)             NSMutableDictionary *headers;
```

Comment:
Using copy for NSMutableDictionary will actually create an immutable copy. If I remember correctly, Eugene removed this line because it was triggering a warning.